### PR TITLE
feat(frontend): add sol worker types

### PR DESCRIPTION
**Pull request by loki344**
#Note

review first:  https://github.com/dfinity/oisy-wallet/pull/4033

# Motivation

We add types for sol wallet workers

# Changes

- add types
- add messages

### CODE REVIEW

**Comment by loki344** [comment]
review https://github.com/dfinity/oisy-wallet/pull/4033 first

File: src/frontend/src/lib/schema/post-message.schema.ts [hunk 1]
```diff
@@ -97,11 +108,14 @@ export const PostMessageResponseSchema = z.enum([
...
 	'syncIcrcWallet',
 	'syncBtcWallet',
+	'syncSolWallet',
 	'syncIcpWalletError',
 	'syncIcrcWalletError',
 	'syncBtcWalletError',
+	'syncSolWalletError',
 	'syncIcpWalletCleanUp',
 	'syncIcrcWalletCleanUp',
+	'syncSolWalletError',
```
Comment (DenysKarmazynDFINITY) [inline src/frontend/src/lib/schema/post-message.schema.ts:118]: `syncSolWalletError` is already there, I think this one was supposed to be `syncSolWalletCleanUp`

Comment (loki344) [reply src/frontend/src/lib/schema/post-message.schema.ts:118]: ah, no i don't think cleanup is used atm. removed duplicate

**Review by DenysKarmazynDFINITY — APPROVED** [review]
LGTM, thanks

**Pull request merged**